packages/frametricks/init.lua

```diff
@@ -260,24 +260,25 @@ We just issued a \autodoc:command{\breakframevertical} command at the start of t
 As you can see, the current frame is called \code{\script{SILE.typesetter:typeset(SILE.typesetter.frame.id)}} and now begins at the start of the paragraph.
 
 Similarly, the \autodoc:command{\breakframehorizontal} command breaks the frame in two into a left and a right frame.
-The command takes an optional argument \autodoc:parameter{offset=<dimension>}, specifying where on the line the frame should be split.
-If it is not supplied, the frame is split at the current position in the line.
+The command takes an optional parameter \autodoc:parameter{offset=<dimension>}, specifying where on the line the frame should be split.
+If \autodoc:parameter{offset} is not supplied, the frame is split at the current position in the line.
 
 The command \autodoc:command{\shiftframeedge} allows you to reposition the current frame left or right.
 It takes \autodoc:parameter{left} and/or \autodoc:parameter{right} parameters, which can be positive or negative dimensions.
 It should only be used at the top of a frame, as it reinitializes the typesetter object.
 
 Combining all of these commands, the \autodoc:command{\float} command breaks the current frame, creates a small frame to hold a floating object, flows text into the surrounding frame, and then, once text has descended past the floating object, moves the frame back into place again.
 It takes two optional parameters, \autodoc:parameter{bottomboundary=<dimension>} and/or \autodoc:parameter{rightboundary=<dimension>}, which open up additional space around the frame.
-At the start of this paragraph, I issued the command
 
-\begin[type=autodoc:codeblock]{raw}
-\float[bottomboundary=5pt]{\font[size=60pt]{C}}.
-\end{raw}
+% At the start of this paragraph, I issued the command
```

Comment:
Oh well noted, regarding commenting this out. I remember this was used in some older version of the manual. I wonder why it is no longer the case (on purpose, or just by mistake?)

Reply:
I tried it, and it just didn't work very well. It seemed a bad implementation of the `dropcaps` package. (Which is what it says in the docs, too.)

Reply:
Yes, one of the proof-of-concept usages for `\float` was for dropcaps, but we have a *much* more robust package for that now and using this as an example was always terrible. There are still use cases for this, but due to limitations they don't involve contiguous paragraphs flowing around the float.